### PR TITLE
sc: Introduce bridgepeer_mock and test cases of MainBridge

### PR DESCRIPTION
**Pull request by aidan-kwon**
## Proposed changes

Introduce followings
- `MockBridgePeer `to test MainBridge efficiently.
- `testBlockChain` to generate a test BlockChain with initial values.
- `TestMainBridge_handle` to test `handle` function of MainBridge.
- `TestMainBridge_SendRPCResponseData` to test `SendRPCResponseData` function of MainBridge

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: node/sc/mainbridge_test.go [hunk 1]
```diff
@@ -195,3 +239,113 @@ func TestMainBridge_handleMsg(t *testing.T) {
...
+	// Variables will be used as return values of mockBridgePeer
+	key, _ := crypto.GenerateKey()
+	nodeID := discover.PubkeyID(&key.PublicKey)
+	peer := p2p.NewPeer(nodeID, "name", []p2p.Cap{})
+	peerID := peer.ID()
+	bridgePeerID := fmt.Sprintf("%x", peerID[:8])
+	pipe, _ := p2p.MsgPipe()
+
+	// mockBridgePeer mocks BridgePeer
+	mockCtrl := gomock.NewController(t)
```
Comment (ehnuje) [inline node/sc/mainbridge_test.go]: You should call `mockCtrl.Finish()` at the end of the test or `defer mockCtrl.Finish()` to indicate the end of mocking test. If it is not called, test does not fail even if it does not meet the expectations.

File: node/sc/mainbridge_test.go [hunk 2]
```diff
@@ -195,3 +239,113 @@ func TestMainBridge_handleMsg(t *testing.T) {
...
+
+	// Test data used as a parameter of SendResponseRPC function
+	data := []byte{0x11, 0x22, 0x33}
+
+	// mockBridgePeer mocks BridgePeer
+	mockCtrl := gomock.NewController(t)
+	mockBridgePeer := NewMockBridgePeer(mockCtrl)
+
+	// Add mockBridgePeer as a peer of `mBridge.BridgePeerSet`
+	mBridge.BridgePeerSet().peers["testID"] = mockBridgePeer
```
Comment (ehnuje) [inline node/sc/mainbridge_test.go]: How about calling `bridgePeerSet.Register`?

File: node/sc/mainbridge_test.go (hunk 1 above)
Comment (aidan-kwon) [reply node/sc/mainbridge_test.go]: Thank you for the notice! I added it.

File: node/sc/mainbridge_test.go (hunk 2 above)
Comment (aidan-kwon) [reply node/sc/mainbridge_test.go]: Good suggestion. I changed the code to use it.

**Pull request merged**